node/util/types.ts

```diff
@@ -0,0 +1 @@
+export { default } from "./types.mjs";
```

Comment:
It feels strange to have both `types.ts` and `types.mjs`. Can you rename `types.mjs` to `types.ts` instead of creating this new file?